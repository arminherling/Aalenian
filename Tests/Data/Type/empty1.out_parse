TypeDefinitionStatement: {
    NameExpression: Empty
    BlockNode(0): {
    }
}
