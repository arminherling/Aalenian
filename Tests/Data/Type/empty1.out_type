TypedTypeDefinitionStatement: {
    TypeKind: type
    TypeName: Empty
    Fields(0): {
    }
    Methods(0): {
    }
}
