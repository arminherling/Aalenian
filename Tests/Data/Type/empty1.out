TypeDefinitionStatement: {
    Name: Empty
    Block(0): {
    }
}
